SpectraLogic.EscapePodClient/Runtime/INetwork.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using SpectraLogic.EscapePodClient.Calls;
+
+namespace SpectraLogic.EscapePodClient.Runtime
+{
+    public interface INetwork
+    {
+        INetwork WithProxy(Uri proxy);
```

Comment:
Will this call return a new INetwork instance that is then proxied, or will it modify the INetwork instance it was called on?

Reply:
Looking at the implementation later, it looks like it modifies it.  Rather than having it modify the existing INetwork instance, it should be passed into the INetwork instance via a builder.